docs/gremlin_tutorials.md

```diff
@@ -0,0 +1,731 @@
+# Tutorials for Gremlin Users
+
+## Introduction
+This documentation guides you how to work with the gremlin graph traversal language in GraphScope. On the one hand we retain the original syntax of most steps from the standard gremlin, on the other hand the usages of some steps are further extended to denote more complex situations in real-world scenarios.
+
```

Comment:
Since this file is very long, It would be better if we could add an index here.
e.g., 
- Supported Steps
- Limitations

and link them to corresponding sections.